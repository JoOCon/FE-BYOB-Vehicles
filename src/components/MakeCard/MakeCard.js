import React from 'react';
import './MakeCard.css';

const MakeCard = ({ Make, Manufacturer }) => (
  <article className="make-card-cont">
    <h2>{ Make}</h2>
    <h2>{ Manufacturer }</h2>
  </article>
);

export default MakeCard;
