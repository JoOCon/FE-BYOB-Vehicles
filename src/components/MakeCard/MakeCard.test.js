import React from 'react';
import { shallow } from 'enzyme';
import MakeCard from './MakeCard';

describe('MakeCard', () => {
  it('should match snapshot with makes', () => {
    const wrapper = shallow(<MakeCard />);

    expect(wrapper).toMatchSnapshot();
  });
});
