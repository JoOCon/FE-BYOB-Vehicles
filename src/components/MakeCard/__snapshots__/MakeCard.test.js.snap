// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MakeCard should match snapshot with makes 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MakeCard />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2 />,
        <h2 />,
      ],
      "className": "make-card-cont",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": "h2",
      },
    ],
    "type": "article",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2 />,
          <h2 />,
        ],
        "className": "make-card-cont",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": "h2",
        },
      ],
      "type": "article",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
